config/deploy/production.rb

```diff
@@ -0,0 +1,57 @@
+# server-based syntax
+# ======================
+# Defines a single server with a list of roles and multiple properties.
+# You can define all roles on a single server, or split them:
+
+# server "train.nzoi.org.nz", user: "cap_deploy", roles: %w[app db web]
+server "faketrain", user: "cap_deploy", roles: %w[app db web]
```

Comment:
Is this just from debugging?

Reply:
yeah, this needs some decisions - we could use a new user, or make some changes to the existing deploy user, and i wanted to make sure that if this got accidentally merged it couldn't do any damage.